### ISSUE TITLE
Error while launching: Failed to execute script steammetaeditor

### ISSUE THREAD

**Issue by AadiSahni** [tralph3/Steam-Metadata-Editor#11]
I launch steammetaeditor.exe and it shows the error: Failed to execute script steammetainstaller. I installed it through  github and I am on Windows 10 20H2

**Comment by AadiSahni** [comment]
I run it, a cmd window pops up, and for a very small time, another windows and then it exits.

**Comment by cloudhaacker** [comment]
hey tralph3 I wanted to try this out too - I did the same thing initially and found this post - when I try to run this with the command line I get the following:

E:\Steam\Steam Metadata Editor>python steammetadataeditor.py
Traceback (most recent call last):
  File "steammetadataeditor.py", line 1531, in <module>
    mainWindow = MainWindow(silent=args.silent)
  File "steammetadataeditor.py", line 93, in __init__
    self.create_main_window()
  File "steammetadataeditor.py", line 307, in create_main_window
    self.mark_installed_games()
  File "steammetadataeditor.py", line 323, in mark_installed_games
    libraries = self.parse_library_folders(libraries.read())
  File "steammetadataeditor.py", line 352, in parse_library_folders
    del lbryDict['TimeNextStatsReport']
KeyError: 'TimeNextStatsReport'

**Comment by tralph3** [comment]
I openly admit this program is a mess of spaghetti code that I should be ashamed of. As such I began a rewrite a couple of days ago with the knowledge I acquired since I wrote it. However... it's going to take a while so, in the meantime, I pushed a quick patch that hopefully fixes this issue.

Since I can't reproduce it, I can't really test it, but in theory it should work now, try to download the latest commit.
